WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -138,24 +141,6 @@ private extension SupportTableViewController {
 
     func tableViewModel() -> ImmuTable {
```

Comment:
With these changes in this PR I think it would be clearer if we rename this function name to be `wpTableViewModel()`?  And also consequentially as a result should `oldTableViewModel()` be renamed to `jpTableViewModel()` and the `TODO` be removed?

Reply:
Good idea! I'll update this.